server/src/main/java/org/elasticsearch/common/util/ByteUtils.java

```diff
@@ -85,45 +82,4 @@ public static float readFloatLE(byte[] arr, int offset) {
         return Float.intBitsToFloat(readIntLE(arr, offset));
     }
 
-    /** Same as DataOutput#writeVLong but accepts negative values (written on 9 bytes). */
```

Comment:
These two methods were unused everywhere except for tests, so I removed them and the tests as well.